android/quest/src/main/assets/configurations/view/patient_register_row_view_configurations.json

```diff
@@ -0,0 +1,37 @@
+[
```

Comment:
is there her only for testing? does this come from the server in the built version?

Reply:
This was done purposefully for testing while the scoping on the server side was in progress. The configs will be downloaded from the server.